src/test/java/org/jitsi/meet/test/MalleusJitsificus.java

```diff
@@ -241,15 +239,15 @@ public void testMain(
         protected final boolean muteVideo;
         protected final boolean muteAudio;
         protected final String region;
+        protected final boolean enableFailureDetection;
 
         WebParticipant participant;
-        public int failureTolerance;
         private String bridge;
 
-        public ParticipantThread(
+        public MalleusTask(
             int i, JitsiMeetUrl url, long durationMs, long joinDelayMs,
             boolean muteVideo, boolean muteAudio, String region,
-            int initialFailureTolerance)
+            boolean enableFailureDetection)
```

Comment:
The boolean that Jonathan had introduced in an earlier version of this class was good enough.